contracts/JBTiered721Delegate.sol

```diff
@@ -762,4 +762,21 @@ contract JBTiered721Delegate is IJBTiered721Delegate, JB721Delegate, Ownable {
     _tokenId;
     _tier;
   }
+
+  /**
+    @notice Return royalty info
```

Comment:
Maybe a bool "useRoyalty", so owners could still use their reserved rate without being force to take royalties?

Reply:
(maybe in fc metadata or smth, so it can't be changed)

Reply:
@drgorillamd made it optional with `JBFundingCycleMetadata.metadata`  have another look when you get time